site/profile/manifests/core/common.pp

```diff
@@ -1,4 +1,13 @@
-class profile::core::common {
+# @summary
+#   Common functionality needed by standard nodes.
+#
+# @param collect_metrics
+#   Enable or disable metrics collection. Metrics collection may be disabled on development
+#   nodes, nodes that don't have uptime requirements, or nodes that should only have minimal
+#   software load.
+class profile::core::common(
+  Boolean $collect_metrics = true,
```

Comment:
I would prefer this to be a direct include of the `profile::core::telegraf` class in hiera as it's one line in hiera either way and direct inclusion involves less indirection. However, I can live with it.

Reply:
Let's break this out into a conversation on classification and roles and profiles, are you available to discuss this at tomorrow's Configuration Management meeting?